registry/modules/upload-to-s3/utils/errors/storeFileS3.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpError } from "@/modules/shared/utils/errors/HttpError.js";
+
+export const fileUploadFailed = (errorCode?: string) => {
+  return new HttpError(
+    "FileUploadFailed",
+    `There was an error uploading your file. ${
+      errorCode && "Error code: " + errorCode
+    }`,
+    400
+  );
+};
+
+export const s3UploadFailed = () => {
+  return new HttpError(
+    "S3FileUploadFailed",
+    `There was a problem when uploading your file to your S3 storage. Please check logs and try again`,
```

Comment:
`Please check logs and try again`

The end customer will see this error. The end user will not be able to check logs.